blockly-core/tests/modal_editors_test.js

```diff
@@ -250,6 +250,31 @@ function test_functionEditor_deleteParam() {
   assertNotContains('Parameter deleted', 'x', paramsUsed);
   assertContains('Still has other parameter', 'y', paramsUsed);
 
+
+  cleanupFunctionEditor();
+  goog.dom.removeNode(container);
+}
+
+function test_functionEditor_useSimpleDialogForParamDeletion() {
+  var container = Blockly.Test.initializeBlockSpaceEditor();
+  initializeFunctionEditor(PROCEDURE_WITH_PARAM);
+  openFunctionEditor('procedure with param 1');
+  var dialogCreated = false;
+  Blockly.customSimpleDialog = function(config) {
```

Comment:
I cannot remember; is Blockly reinitialized between tests? If not, overriding this function has the potential to cause problems in tests executed after this one.